### DISCUSSION TITLE
Table for new Admin View is not inserted

### DISCUSSION THREAD

**Discussion by LordRazen** [joomengine/Joomla-Component-Builder#677] [Q&A] [answered]
I added a new Admin View to my component, but the table is not added while installation. 

I checked out the script.php and found the install() method empty. 

There's an install and an update part in the postflight() method, but it looks more like an update to the values to me. 

The empty install() method is bothering me. It's not generated, but why?

**Comment by Llewellynvdm** [comment, marked as answer]
Please watch the tutorials... these specifically will help.
- Automated database updates in Joomla during development of a component - https://www.youtube.com/watch?v=zN2M15fzf_M
- How to Fix SQL Error in JCB - https://www.youtube.com/watch?v=ZxC6z701ixA
